src/software/backend/input/network/BUILD

```diff
@@ -0,0 +1,67 @@
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "ssl_gamecontroller_client",
+    srcs = ["ssl_gamecontroller_client.cpp"],
+    hdrs = ["ssl_gamecontroller_client.h"],
+    #    visibility = ["//visibility:private"],
```

Comment:
Hmm, should we still have these? I don't us using it outside this file anywhere else in this PR? Same goes for below.

Reply:
yeah I think only client needs to be public

Reply:
Ok, I'm fine with either uncommenting or removing these.

Reply:
changed